spec/factories/custom_button_set.rb

```diff
@@ -2,5 +2,13 @@
   factory :custom_button_set do
     sequence(:name)        { |n| "custom_button_set_#{seq_padded_for_sorting(n)}" }
     sequence(:description) { |n| "custom_button_set_#{seq_padded_for_sorting(n)}" }
+
+    transient do
+      children []
+    end
+
+    after(:create) do |custom_button_set, evaluator|
+      custom_button_set.replace_children(evaluator.children)
+    end
```

Comment:
ahh ok, and this does not affect current users of this Factory outside our API ?